.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,16 @@
-### Behavior
+### What did you do? (required. The issue will be **closed** when not provided.)
 
-Write here what's happening and what you're expecting instead of...
+If possible, please provide a recipe to replicate your problem with a minimal
+`vimrc` with all plugins other than vim-go disabled.
```

Comment:
I would personally prefer to drop the "If possible" part here. I don't know about you, but I don't feel like testing stuff with people's huge ~/.vim directories and 37 plugins. Besides, debugging vimrc files is as easy as commenting stuff out, and something anyone can do.

For example:

> Please make sure you're able to replicate your problem with a minimal `vimrc` with all plugins other than vim-go disabled.

Could maybe also link to [this](https://vi.stackexchange.com/q/2003/51); I think there's also a help page with more or less the same info, but I can't seem to find that at the moment.